src/Files.App/Actions/IAction.cs

```diff
@@ -1,11 +1,14 @@
-﻿using System.Threading.Tasks;
+﻿using Files.App.Commands;
+using System.Threading.Tasks;
 
 namespace Files.App.Actions
 {
 	public interface IAction
 	{
 		string Label { get; }
 
+		HotKey HotKey => HotKey.None;
```

Comment:
Could you replace it with `{ get; }`?

Reply:
Isn't it better to avoid assigning in memory since the value is static?

Reply:
It is assigned in the memory anyway. The code above uses **default interface implementation**. I'd suggest replacing it with `{ get; }` and assigning it in the implementation (or not, assuming that `HotKey.None` *is* the default value in which case it'll be set automatically for you)

Reply:
It is modified in the classes, but not in the interface because it generates a compilation error.

Reply:
Yeah, that's why I think it should be assigned in the implementation and not in the interface by setting the default value